sinks/signalfx/signalfx_test.go

```diff
@@ -739,5 +736,9 @@ func TestSignalFxClientByTagUpdater(t *testing.T) {
 	sort.Strings(expectedPerTagClients)
 	sort.Strings(actualPerTagClients)
 
-	require.Equal(t, expectedPerTagClients, actualPerTagClients)
+	// Check that both sets are equal (subsets of each other)
+	// The order might be different due to modular division, but we should always receive the same
+	// three responses
+	require.Subset(t, expectedPerTagClients, actualPerTagClients, "The actual values should be a subset of the expected values")
+	require.Subset(t, actualPerTagClients, actualPerTagClients, "The expected values should be a subset of the actual values")
```

Comment:
This seems unnecessary? The set of actual clients is always going to be a subset of itself. Think you meant to invert the two arguments instead? 

``` go
require.Subset(t, actualPerTagClients, expectedPerTagClients, "The expected values should be a subset of the actual values")
```

Also, I'd suggest making them `assert`s, so that we can see both fail at once.

Reply:
Good catch - yes, the point was to check that they're both subsets of each other (ie the same set).